source/refined-github.ts

```diff
@@ -220,4 +220,4 @@ import './features/list-pr-for-branch';
 import './features/cancel-wiki-edit-button';
 import './features/comment-on-draft-pr-indicator';
 import './features/select-notifications';
-import './features/remove-unused-repo-tabs';
+import './features/clean-repo-tabs';
```

Comment:
This also covers organizations so it should be named… `clean-tabs`? 🤔

Reply:
Its only one tab in organizations. We could drop it altogether. Or make it a separate feature